docs/mathematics/occurrence-of-no.-in-array.mdx

```diff
@@ -0,0 +1,149 @@
+
+In this article, we will see how to count the occurrence of a number in a given set of data. Occurrence or frequency means how many times a number is present in a given set of data.
+
+## Explanation 
+In this, we have to count the number of times a given element is present in an array that means if an element is present 4 times in the array then we have to print 4.
+#### Example 1
+
+<img src="https://i.imgur.com/8Gwk5QG.png" width="500" height="50"/>
+
+```
+Given arr[8]= | 2 | 4 | 3 | 2 | 4 | 5 | 2 | 3 |
+Given number: 3
+So we iterate on whole array 
+And increase the count if current number is equal to 3
+So after traversing the whole array it prints the count=2 
+That means 3 is present 2 times
+
+```
+## Algorithm
+
+For counting occurrence of an element in array, we follow the following algorithm:
+
+* First, we initialize a count variable with 0.
+
+* Iterate on whole array and increase the count by 1 when our current element is equal to given element.
+
+* If element is not present print Invalid.
+
+* Otherwise display the value of count.
+
+## Code
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+  defaultValue="cpp"
+  values={[
+    { label: 'C++', value: 'cpp', },
+    { label: 'Python', value: 'py', },
+  ]
+}>
+<TabItem value="cpp">
+
+```cpp
+#include<bits/stdc++.h>
+using namespace std;
+
+//function for counting occurrence
+int count_occurrence(int x,int a[],int n)
+{
+  //initialize count with -1
+   int count=-1;
+   
+   //iterating on whole array
+   for(int i=0;i<n;i++)
+   {
+     if(a[i]==x)
+     {
+       count++;
+     }
+   }
+   return count;
+}
+
+//main function
+int main()
+{
+  int n;
+
+  //Taking size as input
+  cout<<"Enter the size of array: ";
+  cin>>n;
+  int a[n];
+  cout<<"Enter the elements of array: ";
+
+  //Taking elements of array
+  for(int i=0;i<n;i++)
+  {
+    cin>>a[i];
+  }
+  int x;
+  cout<<"Enter the no. of which you want to count the occurrence: ";
+  cin>>x;
+  //Printing occurrence of given number
+  cout<<count_occurrence(x,a,n)<<endl;
+  return 0;
+}
+
+```
+</TabItem>
+
+<TabItem value="py">
+
+```python   
+def countOccurrences(arr, n, x):
+    res = 0
+    for i in range(n-1):
+        if x == arr[i]:
+            res += 1
+    return res
+  
+# Taking size as input 
+n = int(input("Enter the size of array: "))
+data = []
+
+# Taking array element as input
+d = int(input("Enter the elements of array: "))
+for i in range(n-1):
+    data.append(int(input()))
+x = int(input("Enter the no. of which you want to print the occurrence: "))  
+
+  #Printing the occurrence of given number 
+print(countOccurrences(data,n,x))
+   
+    
+```
+</TabItem>
+</Tabs>
+
+## Sample Input and Output
+
+#### Input :
+
+```
+Enter the size of array: 8
+Enter the elements of array: 2 4 3 2 4 5 2 3
+Enter the no. of which you want to print the occurrence: 3
+
+```
+#### Output :
+
+```
+2
+```
+
+## Complexity Analysis
+ 
+- **Time Complexity:** `O(n)`
```

Comment:
Can we have the statement `where `n` is the number of elements.` after this line.